Modules/Typescript/02_oop_part_1/code/src/Weapon.ts

```diff
@@ -0,0 +1,52 @@
+import { Item } from './Item';
+
+export abstract class Weapon extends Item {
+  baseDamage: number;
+  damageModifier: number;
+  static MODIFIER_CHANGE_RATE: number = 0.05;
+  // sum of those === broken
+  baseDurability: number; // where to set it?
+  durabilityModifier: number // where to set it?
```

Comment:
I think it is okay to set it either via constructor or as a base constant right here in the class